babel.config.js

```diff
@@ -0,0 +1,34 @@
+// This is solely used to support i18next-extract
+// do not put anything in here that is not related to i18next-extract
+
+const common = ["@babel/env"];
+module.exports = {
+  presets: [
+    "@babel/preset-env",
+    "@babel/preset-typescript",
+    "@babel/preset-react",
```

Comment:
This file is a refactor of the SSN babelrc.  Since gp doesn't use CRA, needed to build this up from scratch and use a json export in order to add the overrides.

Reply:
Babel can only really find input files that are children of the folder containing the babel configuration file.  That's why it was put in the top level of the monorepo.  The CLI commands and dependencies thus all moved up too, but maybe they don't need to.